index.bs

```diff
@@ -5274,8 +5274,8 @@ Compute the [softmax](https://en.wikipedia.org/wiki/Softmax_function) values of
 the 2-D input tensor along axis 1.
 <script type=idl>
 partial interface MLGraphBuilder {
-  MLOperand softmax(MLOperand input);
-  MLActivation softmax();
+  MLOperand softmax(MLOperand input, unsigned long axis);
+  MLActivation softmax(unsigned long axis);
```

Comment:
I was wondering that too. It looks like frameworks either don't have a default, or default to -1 (i.e. last dimension). Would a default of 1 be surprising?

Reply:
Fair. Not having a default is safer to avoid transferrable knowledge surprises (e.g. TF defaults to -1), and we could always relax it later, but once added, it's hard to unrelax. I actually *prefer* being explicit here anyway. I was more thinking of transient compat.